kinto_elasticsearch/views.py

```diff
@@ -29,6 +29,13 @@ def search_view(request, **kwargs):
     bucket_id = request.matchdict['bucket_id']
     collection_id = request.matchdict['collection_id']
 
+    # # Limit the number of results to return, based on existing Kinto settings.
```

Comment:
If I understand correctly the goal is to limit the max size of the response but we cannot paginate the response?
Isn't that too restrictive? What are we trying to achieve here?